styles.css

```diff
@@ -831,3 +824,61 @@ ul {
     margin-bottom: 3%;
   }
 }
+
+@media (max-width: 768px) {
+  .hamburger {
+    display: block;
+    cursor: pointer;
+    margin-left: 86%;
+  }
+
+  .hamburger .line {
+    width: 25px;
+    height: 3px;
+    background-color: #67798e;
+    margin: 6px 0;
+    margin-right: 2%;
+  }
+
+  .menu-section {
+    position: absolute;
+    display: flex;
+    flex-direction: column;
+    gap: 5%;
+    left: 0;
+    top: 60px;
+    width: 100%;
+    background-color: #fff;
+    transition: 0.5s;
+    height: 450px;
+    overflow: hidden;
+    visibility: visible;
+    opacity: 0;
+  }
+
+  .menu-section.active {
+    height: 650px;
+    opacity: 1;
+    border-bottom-left-radius: 6px;
+    border-bottom-right-radius: 6px;
+  }
```

Comment:
- [ ] As suggested by the previous reviewer please make sure to add a bottom border-radius to the menu as shown in the below Figma design. 👇
- Figma design:
  ![image](https://github.com/tajulafreen/Portfolio/assets/104968236/fb576254-545e-49f6-ae3e-0582607350c3)
- Your design:
  ![firefox_Ic39Da92LT](https://github.com/tajulafreen/Portfolio/assets/104968236/c0a60c6b-c25e-46a1-bea2-abc9b7ce211b)
- **HINT:** You should use the `border-bottom-left-radius` and `border-bottom-right-radius`

Reply:
Hi @AbelG101 
I was added but it's not working.

Reply:
Hey @tajulafreen, It wasn't working for me. Can you double-check?